src/firefly/js/tables/ui/BasicTableView.jsx

```diff
@@ -284,4 +285,10 @@ function makeColumns ({columns, columnWidths, data, selectable, showUnits, showT
     return colsEl.filter((c) => c);
 }
 
+function getDefaultRenderer(col={}, tbl_id) {
+    if (col.type === 'location' || !isEmpty(col.links)) {
+        return ((props) => <LinkCell {...props} {...{tbl_id, col}}/>);
+    }
 
+    return TextCell;
```

Comment:
I wonder what happen in the UI if there is no link? Is the icon behave the same and will open an empty tab?